Solutions/Recorded Future Identity/Data/Solution_RecordedFutureIdentity.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Name": "Recorded Future Identity",
+  "Author": "Recorded Future Premier Integrations - support@recordedfuture.com",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/RecordedFuture.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "[Recorded Future](https://www.recordedfuture.com/) Identity Intelligence enables security and IT teams to detect identity compromises, for both employees and customers. To do this, Recorded Future automates the collection, analysis, and production of identity intelligence from a vast range of sources. Organizations can incorporate identity intelligence into automated workflows that regularly monitor for compromised credentials and take immediate action with applications such as Azure Active Directory and Microsoft Sentinel.\nThere are many ways organizations can utilize Recorded Future Identity Intelligence; the playbooks in this Solution are just a quick introduction to some of those ways. In particular, these playbooks include several actions that can be coordinated, or used separately. They include:\n1. searches for compromised workforce or external customer users\n2. looking up existing users and saving the compromised user data to a Log file\n3. confirming high risk Azure Active Directory (AAD) users\n4. adding a compromised user to an AAD security group\n\nFor more information, see the [Documentation for this Solution](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Recorded%20Future%20Identity/Playbooks).",
+  "Analytic Rules": [],
```

Comment:
Please remove the analytic rule and workbook fields as we have only playbooks.

Reply:
I'm using the createSolutionV2.ps1 script in the V2 directory and the input file contains 

``` json
  "BasePath": "c:\\github\\Azure-Sentinel\\Solutions\\Recorded Future Identity\\",
  "Version": "2.0.0",
  "Metadata": "SolutionMetadata.json",
```
But the mainTemplate still reports this

``` json
{
  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
  "contentVersion": "1.0.0.0",
  "metadata": {
    "author": "Recorded Future Premier Integrations - support@recordedfuture.com",
    "comments": "Solution template for Recorded Future Identity"
  },
```

Reply:
We have made the changes regarding the package version, and removed analytic rule and workbook fields.

About "removing playbook description from data file as we will have it in Create UI file":
the CreateUI file is generated out of "Solution_RecordedFutureIdentity.json" file, and if we remove the playbook descriptions from Solution_RecordedFutureIdentity.json file - on a next version release, when CreateUI file will be re-generated from Solution_RecordedFutureIdentity.json file - we will "lose" the playbook descriptions (as they wouldn't be present in the Solution_RecordedFutureIdentity.json file, on which CreateUI file is based).

So I don't think we should remove playbook description from Solution_RecordedFutureIdentity.json file.

If you meant something else - please specify more details. Thanks!


Other than that - all changes are done and ready for your review.
Review once again, please!